src/UI/MvePlayer.cpp

```diff
@@ -837,8 +819,10 @@ void MvePlayer::_processChunk()
         return;
     }
 
-    for (auto opcode : *_chunk->opcodes())
+    auto opcodes = _chunk->opcodes();
+    for (unsigned int i=0; i < opcodes->size();i++)
```

Comment:
Be careful, because

`auto opcodes = *_chunk->opcodes();`
will make a copy of the vector.
`const auto& opcodes = *_chunk->opcodes();`
is a preferred way.